pages/accountLists/[accountListId]/reports/goalCalculator/[[...contactId]].page.tsx

```diff
@@ -65,13 +62,13 @@ const GoalCalculatorContent: React.FC<GoalCalculatorContentProps> = ({
         <RightPanelTitle variant="h6">{t('Details')}</RightPanelTitle>
         <IconButton
           size="small"
-          onClick={() => toggleRightPanel()}
+          onClick={closeRightPanel}
```

Comment:
Lets put this in a useMemo or separate component so it doesn't re-create on each render